cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java

```diff
@@ -174,6 +176,8 @@ public TwillRunnableSpecification configure() {
 
   @Override
   public void initialize(TwillContext context) {
+    System.setSecurityManager(new RunnableSecurityManager());
```

Comment:
so we don't need to cache and reset it? Or are we going to use ``RunnableSecurityManager`` for the entire duration of the Runnable starting from ``initialize``, but it only applies to spark programs?